drinfomon.lic

```diff
@@ -939,11 +1097,33 @@ exp_mods_check_time = Time.now - exp_mods_check_interval
 # Waiting until now to silence the script avoids that ambiguity.
 silence_me
 
+# Now that the hooks are registered,
+# do an initial parse of all the info!
+
+unless dead?
+  squelch_info_output = !UserVars.drinfomon_debug
+  fput('info')
+  pause 0.5
```

Comment:
@MahtraDR @asechrest You know, I don't know the purpose of the pauses either. Worth testing with and without them

Reply:
Without pauses it seemed to work better for me, but it's still race condition/latency based, I think.

Reply:
I've removed the pauses, no negative effects "yet", and things are speedier :)